docs/website/site.eleventy.cjs

```diff
@@ -6,17 +6,24 @@ const syntaxHighlight = require('@11ty/eleventy-plugin-syntaxhighlight');
 const slugify = require('slugify');
 const htmlMinifier = require('html-minifier');
 const fs = require('fs');
-const searchFilter = require("./src/site/js/filters/searchFilter.cjs");
+const searchFilter = require("./src/site/scripts/filters/searchFilter.cjs");
+const anchor = require('markdown-it-anchor');
+
+const DEV = process.env.NODE_ENV === 'DEV';
+const jsFolder = DEV ? 'lib' : 'build';
```

Comment:
Why `build` for prod and `lib` for dev?

Reply:
Because we have different output in dev and production mode, in production mode compressed with gzipPlugin